### ISSUE TITLE
V2.x  Remote start playlist stops fppd

### ISSUE THREAD

**Issue by patdelaney** [FalconChristmas/fpp#386]
I am trying to start a playlist on a remote Pi. When I send the command via and event or from the console on a different system, fppd on the remote stops (crashes) I can't see anything in the logs.

Here is the command line call:
/usr/bin/curl "http://192.168.1.236/fppxml.php?command=startPlaylist&playList=LG-Action-N".

Pat

**Comment by DMXMax** [comment]
I am on version 2.6, and have tested this by creating a playlist called LG-Action-N. 
From the chrome browser, I navigated to 
`http://xx.xx.xx.xx/fppxml.php?command=startPlaylist&playList=LG-Action-N` 
The playlist ran as expected.

I ran the following command from cygwin for windows: 
`curl` `'http://10.0.1.22/fppxml.php?command=startPlaylist&playList=LG-Action-N'`

The playlist LG-Action-N played as expected.
